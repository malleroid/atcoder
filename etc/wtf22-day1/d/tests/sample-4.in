20 15
15 19
1 8
6 11
3 11
11 17
6 6
16 20
7 11
11 14
2 19
1 3
7 7
6 19
14 15
15 15
