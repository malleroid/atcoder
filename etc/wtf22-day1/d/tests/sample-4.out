7
11
12
13
14
15
15
15
15
15
15
15
15
15
15
15
15
15
15
15
